### PR TITLE
llvm.sh is now compatible with most Ubuntu-derivative distros

### PR DESCRIPTION
**Pull request by jonnysoe**
Known distros tested:
- Ubuntu 20.04
- Linux Mint
- Zorin OS 16
- Pop!_OS 20.04

### CODE REVIEW

**Comment by jonnysoe** [comment]
Uploaded old and new test.log, no new failures introduced, but we should update the test cases eventually

[test.old.log](https://github.com/opencollab/llvm-jenkins.debian.net/files/8956787/test.old.log)
[test.new.log](https://github.com/opencollab/llvm-jenkins.debian.net/files/8956789/test.new.log)

File: llvm.sh [hunk 1]
```diff
@@ -10,6 +10,20 @@
 
 set -eux
 
+# helper function to add symbolic link to an executable
+# NOTE: this will NOT replace the executable that was added by installation from official APT
+symlink_llvm() {
```
[the remark this answers is not in the document]
Comment (jonnysoe) [reply llvm.sh:15]: Opps, I never knew that, always thought official APT does that so I mirrored it for PPA, I'll get rid of it then

[the remark this answers is not in the document]
Comment (jonnysoe) [reply llvm.sh]: Updated

**Comment by sylvestre** [comment]
Did you test on Debian?

**Comment by jonnysoe** [comment]
> Did you test on Debian?

I only ran for Debian implicitly from the test.sh script in Docker environment, I can load up a Bullseye VM to test it personally

**Comment by sylvestre** [comment]
If you don't mind, that would be great :)

Message ID: 
***@***.***>

**Comment by jonnysoe** [comment]
@sylvestre ,
I took the plunge, installed Debian :)
I've tested on "bullseye", "testing" and "unstable", they all worked great!

**Pull request merged**